### PR TITLE
docs: enable version picker on docs

### PR DESCRIPTION
**Pull request by bencodezen**
Other versions are not available via angular.dev subdomains yet, so this is focused on enabling the UI element. When we're ready with the other versions, another PR will be issued to connect the proper links so users can switch versions.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: Documentation infrastructure

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jessicajaniuk** [comment]
@bencodezen I don't see anything pop up when I click on the menu.

**Comment by JeanMeche** [comment]
@jessicajaniuk We only have v17 ATM :)

**Comment by jessicajaniuk** [comment]
@JeanMeche Yeah, but I still expected something to happen. This might trigger a screen reader with no items in the list. Maybe we should enable this only when we have content to put in the menu.

File: adev/src/app/core/layout/navigation/navigation.component.html [hunk 1]
```diff
@@ -139,19 +139,19 @@
...
               fill="inherit"
             >
               <path d="M480-345 240-585l56-56 184 184 184-184 56 56-240 240Z" />
-            </svg>
+            </svg> -->
           </button>
 
           <ng-template #docsVersionMiniMenu>
-            <ul class="adev-mini-menu adev-version-picker" cdkMenu>
+            <!-- <ul class="adev-mini-menu adev-version-picker" cdkMenu>
```
Comment (JeanMeche) [inline adev/src/app/core/layout/navigation/navigation.component.html:154]: Is this on purpose to hide the versions ?

Comment (bencodezen) [reply adev/src/app/core/layout/navigation/navigation.component.html:154]: @JeanMeche Yup. The subdomains for angular.dev aren't ready so I wanted to reduce confusion with people selecting a version selector that wasn't working yet.

**Comment by bencodezen** [comment]
> @bencodezen I don't see anything pop up when I click on the menu. I feel like we should probably have _something_ present, even if it's "More soon!" or similar. I'm a little worried about what a screen reader would do with this empty list.

At the time I was chatting with the team, the idea was to surface the UI so people get familiar with the version being in the upper left. Since we didn't have any subdomains ready on the angular.dev domain, the idea was to hide any affordances that would imply you can switch versions on this site.

Another alternative is to allow users to redirect to the old subdomain and sites. 

What do you think @jessicajaniuk?

**Comment by jessicajaniuk** [comment]
@bencodezen Could we instead put something in the menu so that it's not empty. Something like "more coming soon" is enough. I think that would resolve the a11y concern. WDYT?

**Comment by bencodezen** [comment]
@jessicajaniuk Works for me. Will update the PR today!

File: adev/src/app/core/services/version-manager.service.ts [hunk 2]
```diff
@@ -10,22 +10,74 @@ import {Injectable, VERSION, computed, signal} from '@angular/core';
...
-  currentVersion: "stable",
-  historicalVersionsLinkPattern: "https://v{{version}}.angular.dev",
+  currentVersion: 'stable',
+  historicalVersionsLinkPattern: 'https://v{{version}}.angular.dev',
   mainVersions: [
     {
-      version: "stable",
-      url: "https://angular.dev"
+      version: 'next',
+      url: 'https://next.angular.io',
```
[the remark this answers is not in the document]
Comment (bencodezen) [reply adev/src/app/core/services/version-manager.service.ts:18]: That's a good point! I'll remove that then and wait until we have a proper `next` subdomain before adding this item back.

**Comment by bencodezen** [comment]
@jessicajaniuk Updates have been made as requested!

**Review by jessicajaniuk — APPROVED** [review]
LGTM!!!

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit a3e51d99a823c57d575e2b4e90804bcbb97d0df2.

**Pull request closed without merging**